src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Loader } from '../Loader/Loader';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  removeTodo:(todoId: number) => void;
+  changeProperty:(todoId: number, property: Partial<Todo>) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  removeTodo,
+  changeProperty,
+}) => {
+  const { id, title, completed } = todo;
+
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames('todo', { completed: todo.completed })}
+      key={todo.id}
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          onChange={() => changeProperty(id, { completed: !completed })}
+        />
+      </label>
+
+      <span
+        data-cy="TodoTitle"
+        className="todo__title"
+      >
+        {title}
+      </span>
+      <button
+        type="button"
+        className="todo__remove"
+        data-cy="TodoDeleteButton"
+        onClick={() => removeTodo(todo.id)}
+      >
+        ×
+      </button>
+
+      <Loader />
```

Comment:
unconditional "sad" loading 😢

Reply:
Thank you, fixed) Previously I implement loader only in 3rd part)